### PR TITLE
Chapter1/homework/5  - Assignment

### PR DESCRIPTION
**Pull request by Joshhortt**
The Assigment is about Accessing Arrays using loops.
[https://github.com/csinn/CSharp-From-Zero-To-Hero/tree/Chapter1/Homework/5](url)
All Functions & Logic should be placed into ArrayOperations.cs.
It should avoid unnecessary nested loops. for the same purpose.
It should pass 'Run all tests' successfully.

**What it does:**
Just Implementing access Array elements using operations with array and loops:

1.Sorts array in ascending order.
2.Reverses elements of an array.
3.Removes last element from an array.
4.Removes first element from an array.
5.Removes element at a given index from an array.
6.Inserts element at the start of an array.
7.Inserts element at the end of an array.
8.Inserts element at specified index of an array.

### CODE REVIEW

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 1]
```diff
@@ -9,28 +9,64 @@ public static class ArrayOperations
...
+            }
         }
 
+
         /// <summary>
         /// Remove last element in array.
         /// </summary>
         /// <param name="array">Input array.</param>
         /// <returns>A new array with the last element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveLast(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:58]: Reuse RemoveAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 2]
```diff
@@ -39,8 +75,16 @@ public static int[] RemoveLast(int[] array)
         /// <returns>A new array with the first element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveFirst(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:77]: Reuse RemoveAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 3]
```diff
@@ -63,8 +127,19 @@ public static int[] RemoveAt(int[] array, int index)
         /// <returns>A new array with element added at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertFirst(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:130]: Reuse InsertAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 4]
```diff
@@ -75,8 +150,19 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:153]: Reuse InsertAt

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 2 above)
Comment (Joshhortt) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:77]: Just made requested changes.

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 4 above)
Comment (Joshhortt) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:153]: Just made requested changes.

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 3 above)
Comment (Joshhortt) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:130]: Just made requested changes.

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 4 above)
Comment (Almantask) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:153]: You should name those changes. AND you should resolve the remarks :smi

**Review by Almantask — APPROVED** [review]

**Pull request closed without merging**